### PR TITLE
US- BRS to change cluster protection level to encrypt and sign for se…

### PR DESCRIPTION
**Pull request by jaiswalricha**
Hi @TarunKrShukla, 

Please help to review the changes.  Thanks

### CODE REVIEW

File: Scripts/RemediationScripts/Remediate-SetClusterProtectionLevelForServiceFabric.ps1 [hunk 1]
```diff
@@ -0,0 +1,731 @@
...
+        $ServiceFabricSkippedWithSingleNode = @()
+
+        Write-Host "Setting ClusterProtectionLevel to EncryptAndSign on Service Fabric(s)." -ForegroundColor $([Constants]::MessageType.Info)
+
+        # Loop through the list of Service Fabric(s) which needs to be remediated.
+        $ServiceFabricWithoutEncryptandSign | ForEach-Object {
+            $ServiceFabric = $_
+            try
+            {
+                if($_.NodeVMCount -lt 3)
```
Comment (TarunKrShukla) [inline Scripts/RemediationScripts/Remediate-SetClusterProtectionLevelForServiceFabric.ps1:405]: Is this should be ($_.NodeVMCount -eq 1)

**Review by TarunKrShukla — COMMENTED** [review]
Please have a look on the comment.

File: Scripts/RemediationScripts/Remediate-SetClusterProtectionLevelForServiceFabric.ps1 [hunk 2]
```diff
@@ -0,0 +1,731 @@
...
+    if ([String]::IsNullOrWhiteSpace($context))
+    {
+        Write-Host $([Constants]::SingleDashLine)
+        Write-Host "Connecting to Azure account..."
+        Connect-AzAccount -Subscription $SubscriptionId -ErrorAction Stop | Out-Null
+        Write-Host "Connected to Azure account." -ForegroundColor $([Constants]::MessageType.Update)
+    }
+
+    # Setting up context for the current Subscription.
+    $context = Set-AzContext -SubscriptionId $SubscriptionId -ErrorAction Stop
```
Comment (TarunKrShukla) [inline Scripts/RemediationScripts/Remediate-SetClusterProtectionLevelForServiceFabric.ps1:215]: If we are doing Connect-AzAccount with SubId in that case we don't need to explicitly set context.

Comment (jaiswalricha) [reply Scripts/RemediationScripts/Remediate-SetClusterProtectionLevelForServiceFabric.ps1:215]: Hi Tarun, its resolved and pushed.

File: Scripts/RemediationScripts/Remediate-SetClusterProtectionLevelForServiceFabric.ps1 (hunk 1 above)
Comment (jaiswalricha) [reply Scripts/RemediationScripts/Remediate-SetClusterProtectionLevelForServiceFabric.ps1]: Hi Tarun, its resolved and pushed.

**Review by TarunKrShukla — APPROVED** [review]
Looks good

**Pull request merged**